docs/usage/configuration-options.md

```diff
@@ -1028,6 +1028,18 @@ If you wish for Renovate to process only select paths in the repository, use `in
 Alternatively, if you need to just _exclude_ certain paths in the repository then consider `ignorePaths` instead.
 If you are more interested in including only certain package managers (e.g. `npm`), then consider `enabledManagers` instead.
 
+## internalChecksFilter
+
+This setting controls whether Renovate controls when and how filtering of internal checks are performed, particularly when multiple versions of the same update type are available.
```

Comment:
```suggestion
This setting determines whether Renovate controls when and how filtering of internal checks are performed, particularly when multiple versions of the same update type are available.
```

I don't like that we use the word "controls" twice in the same sentence. I don't know if a word like "determines" is better?